src/libraries/System.Formats.Cbor/tests/System.Formats.Cbor.Tests.csproj

```diff
@@ -36,18 +33,13 @@
     <Compile Include="Writer\CborWriterTests.TextString.cs" />
     <Compile Include="CoseKeyHelpers.cs" />
   </ItemGroup>
-
-  <ItemGroup Condition="'$(CborPropertyTests)' == 'True'">
-    <Compile Include="PropertyTests\CborPropertyTests.cs" />
-    <Compile Include="PropertyTests\CborRandomGenerators.cs" />
-  </ItemGroup>
-
+  
   <ItemGroup>
     <ProjectReference Include="..\src\System.Formats.Cbor.csproj" />
+    <ProjectReference Include="CborDocument\System.Formats.Cbor.Tests.DataModel.fsproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(CborPropertyTests)' == 'True'">
-    <ProjectReference Include="CborDocument\System.Formats.Cbor.Tests.DataModel.fsproj" />
+  <ItemGroup>
```

Comment:
nit: merge both packagereference and projectreference items together. We usually have them in the same itemgroup.